GOVERNANCE.md

```diff
@@ -0,0 +1,74 @@
+# Eclipse iceoryx governance
+
+Eclipse iceoryx is governed democratically by its maintainers under the umbrella of [OpenADx](https://openadx.eclipse.org/).
+This document describes the processes additionally to the [Eclipse process](https://www.eclipse.org/projects/handbook/).
+
+## Maintainers
+
+An up to date list of the maintainers can be found at the [Eclipse project page](https://projects.eclipse.org/projects/technology.iceoryx/who).
+
+## Bi-weekly meetup
+
+Bi-weekly every Thursday for 1 hour. Everyone is welcome to join.
+
+```
+08:00 PST (GMT - 8)
+16:00 GMT
+17:00 CET (GMT + 1)
+21:30 IST (GMT + 5.5)
+23:00 CST (GMT + 7)
```

Comment:
Maybe a little bit late for India and especially China. What do you think about having this 1 hour earlier? ... and yes, I will complain about this once I participate from CA ;)

Reply:
As iceoryx developers are currently mainly based in the US and Europe, I would stick to the proposed time. There is no perfect solution here. I guess the Zoom link will be usable at any time. So in case it is too late for someone in APAC, we can schedule an extra meeting.

Reply:
Could we do this on wednesday?! We only have meetings on wednesday and on thursday we have every second week our sprint switch.
I would strongly prefer to concentrate those meetings on one day and not to spread them over the week.

Reply:
Nope, sorry :-( Wednesday is not possible for @budrus and Tuesday is not possible for @ariexi. I would prefer to keep Monday and Friday free, as folks often take vacation on these days.

Reply:
While I'd also like to have it on Wednesday, we would have to do it earlier than GMT + 1. If Thursday is problematic because of the sprint switch, maybe Monday is an option.

Reply:
@mossmaurice, @budrus: for security reasons, I am asked not to publish the password for the meeting in the internet ...
So my current solution will be a registration to the event, means one for the 1st Thursday, one for the 3rd Thursday (at the end 2 registrations) with an automatic response which provides the necessary details as well as the calendar invite. The link will be vaild for 20 (zoom limit) meetings each.
Will that be ok?

Reply:
@mossmaurice, @budrus: One further question, should I refer to https://github.com/eclipse/iceoryx as an "event site"?
Thx!

Reply:
@ariexi Yeah, I think that's a good idea.

Reply:
> @mossmaurice, @budrus: for security reasons, I am asked not to publish the password for the meeting in the internet ...
> So my current solution will be a registration to the event, means one for the 1st Thursday, one for the 3rd Thursday (at the end 2 registrations) with an automatic response which provides the necessary details as well as the calendar invite. The link will be vaild for 20 (zoom limit) meetings each.
> Will that be ok?

@ariexi @budrus @mossmaurice with this restrictions on zoom, wouldn't it be better to just use https://meet.google.com or https://meet.jit.si if privacy is a concern? We could just publish a link and everybody is welcome to join

Reply:
The idea was to use Zoom in order to stay close to Eclipse. All the other Eclipse Working Groups are using it and it seems to be the preferred infrastructure. I'd say we give it a try. Everyone is still welcome to join :-)

Reply:
I had also a small misunderstanding with the "password" issue, which is now clarified with Eclipse ... ;-)
I will talk with Michael later on ... ;-)

Reply:
You can use following link for both Thursday Meetings: https://eclipse.zoom.us/j/95918504483?pwd=RWM5Y1pkeStKVDZsU09EY1hnclREUT09
First meeting will be on January 7th 2021.

Reply:
I'll add this link in a follow-up pull request.